src/components/item/settings/ItemSettings.tsx

```diff
@@ -183,8 +183,7 @@ const ItemSettings: FC<Props> = ({ item }) => {
         {renderPinSetting()}
         {renderChatSetting()}
         {renderCollapseSetting()}
-        {/* todo: change itemType to app */}
-        {item.type === ItemType.FOLDER && renderResizeSetting()}
+        {item.type === ItemType.APP && renderResizeSetting()}
```

Comment:
Quick question, do you know why this was set to folder?

Reply:
Left over by me from testing in #514 ... 😅